docs/guides/setup-lando-on-windows-with-wsl-2.md

```diff
@@ -29,41 +29,47 @@ feed:
 
-After installing the kernel update, its time to head to the Microsoft store and install a Linux distro. I am going to choose Ubuntu 18.04 LTS, since it is the most common distro for Lando users and our handy (Hyperdrive)[https://github.com/lando/hyperdrive] supports it.
+1. Open the start menu and search for "Turn Windows features on or off".
+2. In the resulting window, check 'Windows Subsystem for Linux' and 'Virtual Machine Platform' and then click 'ok'. This starts the system installed for WSL and will ask us to restart after it downloads the stuff it needs. This really only gives you the WSL1 setup. To get WSL2...
+3. Download a kernel update to get WSL2 going. see the (Microsoft docs on installing the kernel update)[https://docs.microsoft.com/en-us/windows/wsl/install-win10#step-4---download-the-linux-kernel-update-package].
+4. After you install the kernel update, launch a powershell and run `wsl --set-default-version 2` to default to WSL 2.
+5. Head to the Microsoft store and install a Linux distro. I am going to choose [Ubuntu 18.04 LTS](https://www.microsoft.com/en-us/p/ubuntu-1804-lts/9n9tngvndl3q), since it is the most common distro for Lando users and our handy (Hyperdrive)[https://github.com/lando/hyperdrive] supports it.
 
 ## Jump into Hyperspace
 
-After installing the latest LTS Ubuntu, using Hyperdrive makes the rest extremely easy. We simply lauch a bash shell in the WSL instance, follow the user setup process, and then follow the hyperdrive instructions. 
+After installing the latest LTS Ubuntu, using Hyperdrive makes the rest extremely easy. We simply launch a bash shell in the WSL instance, follow the user setup process, and then follow the Hyperdrive instructions in your terminal.
 
 ## Rules to keep your setup easy and happy
 
-1. **never** run a lando app from within the default directory that WSL drops you into (`/mnt/c/Users/whateveruser`). This is the Windows filesystem mounted into your WSL instance, and it suffers from the filesystem performance issues that this setup avoids, therefore making going through the trouble of setting this up pointless.
-2. You have to start Docker every time you start up the WSL2 instance. I just run `sudo service docker start` but you can probably do some `.bashrc` magic to do this automatically. This is because the WSL2 distros have no init system, so docker doesn't auto start.
+1. **Never** run a Lando app from within the default directory that WSL drops you into (`/mnt/c/Users/whateveruser`). This is the Windows filesystem mounted into your WSL instance, and it suffers from the filesystem performance issues that this setup avoids, therefore making going through the trouble of setting this up pointless.
+2. You have to start Docker every time you start up the WSL2 instance. I just run `sudo service docker start` but you can probably do some `.bashrc` magic to do this automatically. This is because the WSL2 distros have no init system, so Docker doesn't auto start.
```

Comment:
For me docker is also automatically started, if I run `lando start` for the first time after reboot

Reply:
I guess that makes sense, since Lando should know how to initialize Docker from within a standard Ubuntu environment @rabauss. So unless you have a contradictory experience @dustinleblanc, perhaps this warning is unnecessary?

Reply:
It has never worked for me, but it's possible that this has been fixed. I may have just gotten accustomed to doing this from before it worked properly